src/app/courses/course-item.ts

```diff
@@ -0,0 +1,9 @@
+import { Course } from './course';
+
+export class CourseItem implements Course {
```

Comment:
It's ok for now. But keep in mind that we should rethink it in the future.
- the type of implemented interface
- the necessity of this service